eng/templates/official/jobs/build-artifacts.yml

```diff
@@ -0,0 +1,80 @@
+jobs:
+  - job: "Build"
+    displayName: "Build python worker"
+
+    templateContext:
+      outputParentDirectory: $(Build.ArtifactStagingDirectory)
+      outputs:
+        - output: pipelineArtifact
+          targetPath: $(Build.ArtifactStagingDirectory)
+          artifactName: "drop"
+        - output: nuget
+          condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/dev'), eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true))
+          useDotNetTask: false
+          packagesToPush: "$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg"
+          packageParentPath: "$(Build.ArtifactStagingDirectory)"
+          publishVstsFeed: "e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df"
+          nuGetFeedType: "internal"
```

Comment:
is this required for public build as we share the template between both?

Reply:
Removed

Reply:
do we also need L16 for publishVstsFeed?

Reply:
Updated. Publishing is only done in the official build, not for public build.